2024-11-03 11:14:26,393 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:18:18,214 - INFO - Бот успешно запущен
2024-11-03 11:18:24,449 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:24:31,808 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db
2024-11-03 11:27:53,947 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:29:05,737 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:33:44,774 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:34:04,954 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:34:04,970 - INFO - Бот успешно запущен
2024-11-03 11:34:19,322 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:34:19,335 - INFO - Бот успешно запущен
2024-11-03 11:35:33,592 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:51:38,747 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:51:38,757 - INFO - Бот успешно запущен
2024-11-03 11:51:42,772 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:51:42,982 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: unsupported parse_mode
2024-11-03 11:51:42,983 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\__init__.py", line 1235, in __threaded_polling
    self.worker_pool.raise_exceptions()
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\util.py", line 150, in raise_exceptions
    raise self.exception_info
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\util.py", line 93, in run
    task(*args, **kwargs)
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\__init__.py", line 8945, in _run_middlewares_and_handler
    result = handler['function'](message)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\webapp\bots-agregator\main.py", line 31, in send_welcome
    bot.send_message(chat_id, message, "Добро пожаловать!")
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\__init__.py", line 1775, in send_message
    apihelper.send_message(
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\apihelper.py", line 273, in send_message
    return _make_request(token, method_url, params=payload, method='post')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: unsupported parse_mode

2024-11-03 11:52:27,225 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:52:27,250 - INFO - Бот успешно запущен
2024-11-03 11:52:31,437 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:55:03,419 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:55:03,429 - INFO - Бот успешно запущен
2024-11-03 11:55:06,930 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:55:07,137 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: can't parse inline keyboard button: Text buttons are unallowed in the inline keyboard
2024-11-03 11:55:07,137 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\__init__.py", line 1235, in __threaded_polling
    self.worker_pool.raise_exceptions()
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\util.py", line 150, in raise_exceptions
    raise self.exception_info
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\util.py", line 93, in run
    task(*args, **kwargs)
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\__init__.py", line 8945, in _run_middlewares_and_handler
    result = handler['function'](message)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\webapp\bots-agregator\main.py", line 31, in send_welcome
    bot.send_message(chat_id, "Добро пожаловать!", reply_markup=keyboard)
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\__init__.py", line 1775, in send_message
    apihelper.send_message(
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\apihelper.py", line 273, in send_message
    return _make_request(token, method_url, params=payload, method='post')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\webapp\bots-agregator\venv\Lib\site-packages\telebot\apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: can't parse inline keyboard button: Text buttons are unallowed in the inline keyboard

2024-11-03 11:56:26,270 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:56:26,286 - INFO - Бот успешно запущен
2024-11-03 11:56:29,574 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 11:56:49,787 - INFO - Подключение к базе данных по адресу sqlite:///db/data.db?check_same_thread=False
2024-11-03 11:56:49,795 - INFO - Бот успешно запущен
2024-11-03 11:56:52,846 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 13:59:00,085 - INFO - Бот успешно запущен
2024-11-03 14:07:25,246 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 14:07:26,501 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 14:08:36,763 - INFO - Бот успешно запущен
2024-11-03 14:08:44,417 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 14:08:45,538 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 14:09:36,876 - INFO - Бот успешно запущен
2024-11-03 14:09:39,427 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 14:09:40,603 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 14:10:26,104 - INFO - Бот успешно запущен
2024-11-03 14:10:28,732 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 14:10:30,209 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 14:19:09,018 - INFO - Бот успешно запущен
2024-11-03 14:19:29,850 - INFO - Бот успешно запущен
2024-11-03 15:29:50,636 - INFO - Бот успешно запущен
2024-11-03 15:36:20,393 - INFO - Бот успешно запущен
2024-11-03 15:36:38,816 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:36:59,102 - INFO - Бот успешно запущен
2024-11-03 15:37:06,821 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:38:13,470 - INFO - Бот успешно запущен
2024-11-03 15:38:16,394 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:38:17,587 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 15:38:19,917 - INFO - Команда /add_bot от пользователя: 1263818809
2024-11-03 15:38:22,648 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:38:23,795 - INFO - Команда /add_bot от пользователя: 1263818809
2024-11-03 15:38:31,130 - INFO - Команда /add_bot от пользователя: 1263818809
2024-11-03 15:38:51,466 - INFO - Бот успешно запущен
2024-11-03 15:38:53,698 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:40:20,203 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:40:21,700 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 15:40:36,243 - INFO - Бот успешно запущен
2024-11-03 15:40:40,367 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:40:55,879 - INFO - Бот успешно запущен
2024-11-03 15:40:58,199 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:41:59,131 - INFO - Бот успешно запущен
2024-11-03 15:42:03,325 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:42:22,395 - INFO - Бот успешно запущен
2024-11-03 15:42:30,286 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:42:55,225 - INFO - Бот успешно запущен
2024-11-03 15:42:58,796 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 15:43:12,095 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 15:47:27,667 - INFO - Бот успешно запущен
2024-11-03 15:47:33,420 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:49:58,062 - INFO - Бот успешно запущен
2024-11-03 15:50:01,752 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:51:20,267 - INFO - Бот успешно запущен
2024-11-03 15:51:22,917 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:51:23,979 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 15:51:45,443 - INFO - Бот успешно запущен
2024-11-03 15:51:49,254 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:52:11,184 - INFO - Бот успешно запущен
2024-11-03 15:52:13,291 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:53:27,927 - INFO - Бот успешно запущен
2024-11-03 15:53:32,181 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:54:40,286 - INFO - Бот успешно запущен
2024-11-03 15:54:42,878 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:54:43,801 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 15:55:19,610 - INFO - Бот успешно запущен
2024-11-03 15:55:28,088 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:58:04,498 - INFO - Бот успешно запущен
2024-11-03 15:58:18,367 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 15:58:56,692 - INFO - Бот успешно запущен
2024-11-03 15:58:59,237 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 16:03:14,372 - INFO - Бот успешно запущен
2024-11-03 16:03:17,716 - INFO - Команда /start от пользователя: 1263818809
2024-11-03 16:03:34,176 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 16:04:25,828 - INFO - Команда /bot_list от пользователя: 1263818809
2024-11-03 16:04:34,116 - INFO - Бот успешно запущен
2024-11-03 16:04:40,984 - INFO - Команда /bot_list от пользователя: 1263818809
